src/TagLib/Ogg/GroupedComment.cs

```diff
@@ -1378,22 +1414,22 @@ public override string[] PerformersRole
 	/// <seealso cref="Tag.ReplayGainTrackGain" />
 	public override double ReplayGainTrackGain
 	{
-  	get
-  	{
+	get
```

Comment:
This is a good change, but it sadly pollutes the PR. I wouldn't worry about fixing this one though.